pmm/infrastructure/rpm-build.groovy

```diff
@@ -86,9 +86,9 @@ pipeline {
                             secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
                             sh '''
                                 cd build/docker/rpmbuild/
-                                docker build --pull --tag ${IMAGE_REGISTRY}/rpmbuild:3 -f Dockerfile.el9 .
+                                docker build --pull --tag ${IMAGE_REGISTRY}/rpmbuild:ol9 -f Dockerfile.el9 .
                                 aws ecr-public get-login-password --region us-east-1 | docker login -u AWS --password-stdin ${IMAGE_REGISTRY}
-                                docker push ${IMAGE_REGISTRY}/rpmbuild:3
```

Comment:
rpmbuild:3 lives in a separate file now - https://github.com/Percona-Lab/jenkins-pipelines/blob/4b44c06f716aad778bd03e83e549d49960f36083/pmm/infrastructure/rpm-build-3.groovy#L15